docs/content-server-api/contents/hashId/active-entity/get.md

```diff
@@ -0,0 +1,27 @@
+# Get active entity for a content hash
+
+Get the entity id of the deployment associated with a content hash
+
+**URL** : `/contents/:hashId/active-entity`
```

Comment:
Don't forget to update the endpoint here too!

Reply:
Yes, updated